Task/MCU/Floor1_USART_inputdata/Core/Src/main.c

```diff
@@ -128,6 +142,13 @@ void toDoOnAlarm(void)
 }
 #endif //ifdef R_T_C
 
+#ifdef I_W_D_G
+void watchdogFeed(void)
```

Comment:
Why do we need to create separate function? as we can directly use HAL_IWDG_Refresh in application code.

Reply:
We can directly use HAL_IWDG_Refresh in application code but to make it more readable and other can understand more clearly that, for IWDG timeout it will feed i.e. it will refresh the watchdog using watchdogFeed() function. Therefore I used it.